fido/fido.py

```diff
@@ -297,6 +316,7 @@ def fetch(
     timeout=DEFAULT_TIMEOUT,
     connect_timeout=DEFAULT_CONNECT_TIMEOUT,
     tcp_nodelay=False,
+    gzip_enabled=False,
 ):
     """
     Make an HTTP request.
```

Comment:
Can you add the gzip_enabled kwarg to the docstring for fetch?

One thing that's worth mentioning here IMO is that when the response comes back, there will be a response header that says Content-Encoding is gzip, but really that _was_ the response coding and we already decompressed it. I think this is the right behavior, but worth adding clarifications on how this works to the docstring.

Reply:
Woops, I meant to add it but completely forgot. Yeah that sounds good to me. I think most clients (including browsers) still have the encoding set to gzip after decompression